CClash/Compiler.cs

```diff
@@ -477,8 +478,10 @@ public bool ProcessArguments(string[] args)
 
                         case "/Fo":
                             ObjectTarget = Path.Combine(WorkingDirectory, full.Substring(3));
-                            if (!Path.GetFileName(ObjectTarget).Contains("."))
+                            ObjectTarget = ObjectTarget.Replace('/', '\\');
```

Comment:
I can't see the reason for this.  If this is about coping with cygwin paths that shouldn't really happen here.